### PR TITLE
Analytics integration tests

### PR DESCRIPTION
**Pull request by hsubox76**
Add some simple integration smoke tests to analytics dynamic measurement ID PR. Should this run in CI? Which workflow?

### CODE REVIEW

File: packages/analytics/testing/integration-tests/integration.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    firebase.initializeApp(config);
+    firebase.analytics().logEvent('login');
+    await new Promise(resolve => setTimeout(resolve, 10000));
+    const resources = performance.getEntriesByType('resource');
+    const callsWithEvent = resources.filter(
+      resource =>
+        resource.name.includes('google-analytics.com') &&
+        resource.name.includes('en=login')
+    );
+    expect(callsWithEvent.length).to.be.at.least(1);
```
Comment (Feiyang1) [inline packages/analytics/testing/integration-tests/integration.ts:39]: Should it be exactly once?

Comment (hsubox76) [reply packages/analytics/testing/integration-tests/integration.ts]: Changed.

**Comment by hsubox76** [comment]
> Can you try to include integration tests in the `yarn test` script? I think we can just add them to the normal test run if they runs reasonably fast.

Do you think there's any circumstances where it's bad that the test makes a live network request? Or is it assumed anyone running the test will be online?

**Comment by Feiyang1** [comment]
> > Can you try to include integration tests in the `yarn test` script? I think we can just add them to the normal test run if they runs reasonably fast.
> 
> Do you think there's any circumstances where it's bad that the test makes a live network request? Or is it assumed anyone running the test will be online?

I think it's okay. We can add a script that only runs unit tests to mitigate the concern.

File: packages/analytics/package.json [hunk 2]
```diff
@@ -15,9 +15,10 @@
     "build": "rollup -c",
     "build:deps": "lerna run --scope @firebase/analytics --include-dependencies build",
     "dev": "rollup -c -w",
-    "test": "run-p lint test:browser",
-    "test:ci": "node ../../scripts/run_tests_in_ci.js -s test:browser",
+    "test": "run-p lint test:browser test:integration",
+    "test:ci": "node ../../scripts/run_tests_in_ci.js -s test",
```
Comment (Feiyang1) [inline packages/analytics/package.json:19]: `test:ci` should not run lint. You can create a `test:all` script that doesn't do lint and to be used by `test` and `test:ci`

Comment (hsubox76) [reply packages/analytics/package.json]: Done.

**Pull request merged**